Makefile

```diff
@@ -41,11 +41,13 @@ sass:
 install-test:
 	pip install -q -r requirements/test.txt
 
-install-sys-requirements: install-system install-node
+install-sys-requirements: install-system install-node install-sass
```

Comment:
Hey @efischer19 Indeed, reading over that FAQ, the ruby removal would cause a problem here. However, instead of calling `sudo gem`, there should be steps to instantiate rbenv in the given directory. 

That or the user needs to call `sudo make install-sass` in order to invoke the sudo user when executing the gem step.